src/components/app-wrapper.js

```diff
@@ -56,6 +56,9 @@ import { PARAM_THEME } from '../utils/config-params';
 const lightTheme = createTheme({
     palette: {
         mode: 'light',
+        customButton: {
+            main: '#000000DE',
```

Comment:
Maybe we should use values from the defaultTheme which is simpler : 
`theme.palette.text.primary` for example.
It's possible to define a custom theme color but is it useful ?
```
lightTheme = createTheme(theme, {
  palette: {
    customButtonColor: {
      main: theme.palette.text.primary,
    },
  },
});
```

Reply:
Done